ouroboros-network/src/Ouroboros/Network/BlockFetch/Decision.hs

```diff
@@ -582,7 +590,7 @@ fetchRequestDecision
   -> PeerFetchInFlightLimits
   -> PeerFetchInFlight header
   -> PeerFetchStatus
-  -> FetchDecision [ChainFragment header]
+  -> FetchDecision (CandidateFragments header)
```

Comment:
Or we could have dropped this extra info in or after `prioritisePeerChains`. It's just needed as input to `prioritisePeerChains` not needed thereafter.

Reply:
Indeed, this info is not necessary. I have dropped it in my updated version.